examples/node/pdf2svg.js

```diff
@@ -39,14 +39,15 @@ function writeToFile(svgdump, pageNum) {
 function getFileNameFromPath(path) {
   var index = path.lastIndexOf('/');
   var extIndex = path.lastIndexOf('.');
-  return path.substring(index , extIndex);
+  return path.substring(index, extIndex);
 }
 
 // Will be using promises to load document, pages and misc data instead of
 // callback.
 pdfjsLib.getDocument({
   data: data,
-  disableNativeImageDecoder: true,
+  // Try to export JPEG directly if don't need canvas to decode images
```

Comment:
Nit: This comment reads a little strangely, I'd suggest e.g. `Try to export JPEG images directly if they don't need any further processing.` instead.